packages/babel-register/src/node.js

```diff
@@ -77,7 +79,13 @@ function compile(code, filename) {
     result.mtime = mtime(filename);
   }
 
-  maps[filename] = result.map;
+  if (result.map) {
+    // Install sourceMapSupport on receiving the first map
+    if (!Object.keys(maps).length) {
```

Comment:
Can just be `.length > 0` right?

Reply:
No, actually what I meant was `.length == 0` to run the installation only once, when nothing has been added to the `maps` yet.
I agree that the above syntax is not always easily readable, but it's the shortest check for object emptiness that I know of.

Reply:
ah that's why I had that comment below I suppose 😛